### PR TITLE
Add income precheck endpoint

### PR DESCRIPTION
**Pull request by tokafish**
This adds the `/income/verification/precheck` endpoint to the collection

### CODE REVIEW

File: Plaid API Endpoints Collection.json [hunk 1]
```diff
@@ -5641,7 +5676,7 @@
 							},
 							"response": [
 								{
-									"name": "Error response",
```
Comment (tokafish) [inline Plaid API Endpoints Collection.json:5644]: I'm not sure why some of these changed? I simply imported the existing collection and then exported it after adding the new endpoint.

Comment (tokafish) [reply Plaid API Endpoints Collection.json:5644]: (it might just be a re-ordering)

**Review by phoenixy1 — COMMENTED** [review]
two items to check out!

1) I assume you've reimported this and tested it and it works?
2) Whenever you update the postman collection you will need to update the run in postman button link in the readme to point to the new collection; it doesn't by default update in place with our current settings. This is super duper easy and there is even a feature within postman that will generate the button and the link for you. follow the instructions for the static button as described here: https://learning.postman.com/docs/publishing-your-api/run-in-postman/creating-run-button/

**Comment by phoenixy1** [comment]
I have incorporated these changes into my larger postman PR https://github.com/plaid/plaid-postman/pull/64 so closing this out

File: Plaid API Endpoints Collection.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "99adf38b-18e6-40ce-b032-ce23071fc46c",
+		"_postman_id": "52c18bea-b9c5-4c5e-a38b-4c92113363c7",
```
Comment (antonyggvzvmnxxcx) [inline Plaid API Endpoints Collection.json:3]: Antony downey

**Pull request closed without merging**